.github/workflows/TagBot.yml

```
@@ -0,0 +1,12 @@
+name: TagBot
+on:
+  schedule:
+    - cron: 0 0 * * *
+jobs:
+  TagBot:
```

```suggestion
name: TagBot
on:
  issue_comment:
    types:
      - created
  workflow_dispatch:
jobs:
  TagBot:
    if: github.event_name == 'workflow_dispatch' || github.actor == 'JuliaTagBot'
```
That's the latest recommendation, to have it run only on specific triggers, and not daily.